src/main/java/com/exadel/sober/repositories/PromiseRepository.java

```diff
@@ -1,8 +1,22 @@
 package com.exadel.sober.repositories;
 
 import com.exadel.sober.models.Promise;
-import org.springframework.data.repository.CrudRepository;
+import com.exadel.sober.models.Reason;
+import org.springframework.jdbc.core.RowMapper;
 
-public interface PromiseRepository extends CrudRepository<Promise, Integer> {
+import java.sql.ResultSet;
+import java.util.List;
 
+public interface PromiseRepository {
+    RowMapper<Promise> ROW_MAPPER_PROMISE = (ResultSet resultSet, int rowNum) -> {
```

Comment:
it's interface - it must not have any method implementation or variables. It can have default method but it's not yours case